apps/acrs/translations/src/en/pages.json

```diff
@@ -64,7 +64,7 @@
     "header": "SKELETON: /complete-as-referrer"
   },
   "full-name": {
-    "header": "SKELETON: /full-name"
+    "header": "What is your full name?"
```

Comment:
If you use `isPageHeading` you can remove the `full-name` block from the main part of pages.json.